test/client-unit/services/StockService.spec.js

```diff
@@ -0,0 +1,126 @@
+/* global inject, expect */
+
+describe('StockService', StockServiceTests);
```

Comment:
This is a good first attempt!

I propose you make a few lots/inventory items in the [MockDataService](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/test/client-unit/mocks/data.js) (if they don't exist already) and use those in your tests.  That way, this code can focus on the test itself, and spend less time setting up the tests.